info/v1/container.go

```diff
@@ -848,6 +848,25 @@ type PerfStat struct {
 	Cpu int `json:"cpu"`
 }
 
+type MemoryBandwidthMonitoringStats struct {
+	// The 'mbm_total_bytes'.
```

Comment:
This looks like a reference to something.  Can you include a link to where this comes from?

Reply:
Done.